enterprise/models.py

```diff
@@ -689,6 +689,16 @@ def enroll(self, course_run_id, mode):
                 )
             )
 
+    def unenroll(self, course_run_id):
+        """
+        Unenroll a user from a course track.
+        """
+        enrollment_api_client = EnrollmentApiClient()
+        if enrollment_api_client.unenroll_user_from_course(self.username, course_run_id):
+            EnterpriseCourseEnrollment.objects.filter(enterprise_customer_user=self, course_id=course_run_id).delete()
```

Comment:
my only concern with this is if for some reason we had an enterprise course enrollment record but no enrollment record. that state shouldn't be possible to get into, so this is probably fine.